CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# master
```

Comment:
Can you change it to `2.1.0.1`  and also do a corresponding bump in the `package.yaml` file ? Thanks!

Reply:
Sure, `2.1.1` exists on hackage already but I can make it `2.1.1.1`. Did notice there's no entry for `2.1.1` changes in `CHANGELOG` though.

Reply:
Bumped.

Reply:
Thanks for checking, you're right!